src/middleware/socket.js

```diff
@@ -219,7 +168,6 @@ const chatMiddleware = store => next => action => {
 
   socket.on(events.chatHistory, ({ history, errors, region }) => {
     store.dispatch(resetIsLoading());
-    store.dispatch(resetIncoming());
     store.dispatch(setRegion(region));
 
     const { config, meta, chat } = store.getState();
```

Comment:
@IlievaDayana You could destruct 

`const { config, meta: { cid, region }, chat }`

and use it below:

`
handleMessageSending({
      user_id: cid,
      region
 });
`